src/utils/mod.rs

```diff
@@ -82,6 +84,74 @@ pub fn title(res: &mut String, s: &str) {
     writeln!(res).unwrap();
 }
 
+/// Field element helpers
+///   Unless otherwise stated everything is in little-endian byte order.
+pub trait FieldHelpers<F> {
```

Comment:
these have moved here https://github.com/o1-labs/proof-systems/blob/c1803e0a480172c193ba67c826110980303a172f/utils/src/field_helpers.rs#L72 so no need to copy it in noname